root/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Views/Crud1/Index.cshtml

```diff
@@ -0,0 +1,118 @@
+﻿@*
+    //**********************************************************************************
+    //* サンプル アプリ・モデル
+    //**********************************************************************************
+
+    // テスト用クラスなので、必要に応じて流用 or 削除して下さい。
+
+    //**********************************************************************************
+    //* クラス名        ：Index
+    //* クラス日本語名  ：CRUDサンプル（Html.BeginForm）
+    //*
+    //* 作成日時        ：－
+    //* 作成者          ：－
+    //* 更新履歴        ：－
+    //*
+    //*  日時        更新者            内容
+    //*  ----------  ----------------  -------------------------------------------------
+    //*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
+    //**********************************************************************************
+*@
+
+@using MVC_Sample.Models.ViewModels
+@model CrudViweModel
+
+@{
+    // ページタイトル
+    ViewBag.Title = "CRUDサンプル（Html.BeginForm）";
+
+    // マスターページ
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@* スクリプト・セクション（RenderSection("HeaderScriptsSection", required: true)に出力されます） *@
+@section HeaderScriptsSection{
+    <!-- Head 部の section -->
+}
+
+@* 本文レイアウト（RenderBodyに出力されます） *@
+@* Html.BeginFormは従来の全体方式、Ajax.BeginFormは部分更新方式（UpdateTargetIdの範囲を部分更新） *@
+@using (Html.BeginForm(null, null, FormMethod.Post, new { id = "form1", onsubmit = "Fx_OnSubmit();" }))
+{
+    @Html.AntiForgeryToken()
+
+    <div class="form-inline">
+        <div class="form-group">
```

Comment:
@yusukemaegawa 

- Will you delete the following ?  I am worried about that it may be necessary.
https://github.com/OpenTouryoProject/OpenTouryo/blob/94f4619648a4ee9d68f161f25faaaf9ae03dfa1a/root/programs/C%23/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Views/Crud1/Index.cshtml#L85

- Does this mean that two box level elements are displayed side by side in width of 50% size?
```XML
class="col-xs-6 col-sm-6 col-md-6 col-lg-6"
```
- I found a description that not only form-group and form-control but also control-label description is necessary.
  - Bootstrap3におけるFormに関して - 新人Webエンジニアの記録。  
http://blog.fagai.net/2014/08/06/bootstrap3-form/